### ISSUE TITLE
newsfragments directory not created

### ISSUE THREAD

**Issue by htgoebel** [twisted/towncrier#85]
When running `towncrier --draft` first time, it crashes with:
```
Loading template...
Finding news fragments...
Traceback (most recent call last):
…
  File "/tmp/myvenv/lib/python2.7/site-packages/towncrier/_builder.py", line 46, in find_fragments
    files = os.listdir(section_dir)
OSError: [Errno 2] No such file or directory: '…/PyInstaller/newsfragments'
```

towncrier should not crash with a traceback, but catch the exception and print a nice error message without traceback. Or it should create this directory.

**Comment by hawkowl** [comment]
Thanks for catching this. I'll fix it for next release.

**Comment by altendky** [comment]
So I coded up #298 to treat no newsfragment directory the same as an empty one.  It doesn't seem to me like drafting ought to create source directories?  The issue I see with my PR is if people get confused about what directory is being looked at and towncrier silently looks at the 'wrong' one and claims there is no news.  But I'm not sure this deserves a failure or warning.

@htgoebel, how do you feel about this after these years have passed?  Your original request was to fail or create the directory.

**Comment by htgoebel** [comment]
@altendky "feel"? If you now fixed this, it's fine for me.

**Comment by altendky** [comment]
@htgoebel, I addressed the unclear exception in a way other than you had requested.  I wanted to give you a chance to weigh in on whether my change addressed your concerns satisfactorily.  As is, this will get closed when #298 gets reviewed and merged.
